### PR TITLE
Feature/support rtf

### PR DESCRIPTION
**Pull request by mcnallys**
Handle the insanity that is Platform font names, vs font family vs adobe shortcut names.
This lets us draw richtext to rtf fields correctly.

### CODE REVIEW

File: src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs [hunk 1]
```diff
@@ -370,5 +370,187 @@ public void CreateStyle_IsBold_IsItalic()
 
             Assert.IsTrue(fontStyle == XFontStyle.BoldItalic, "fontStyle should be BoldItalic");
         }
+
+        #region Font name mapping tests
```
Comment (pex-simsr) [inline src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs:374]: Good looking tests.  For maintenance and tracking, these should go into a new Test Class for XFontTests under the Drawing Folder.

Comment (pex-simsr) [reply src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs:374]: Otherwise, you can call the FontHelper.MapFamilyNameToSystemFontName directly

**Review by pex-simsr — CHANGES_REQUESTED** [review]
This breaks the work on Rotation on pages/fields

File: src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs (hunk 1 above)
Comment (mcnallys) [reply src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs:374]: I'll move them to a new test class for xfont, i wanted to call it through xfont to be sure it was calling the method to assign the content font name.

**Review by pex-simsr — COMMENTED** [review]
DefaultFontEncoding still causing UTs to break during automation.  Looks like an issue running all Async

**Review by pex-simsr — APPROVED** [review]

**Pull request merged**